addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2525,7 +2531,8 @@ const Wysiwyg = Widget.extend({
         this.odooEditor.collaborationSetClientId(this._currentClientId);
         this.setValue(value);
         this.odooEditor.historyReset();
-        this.ptp.notifyAllClients('ptp_join');
+        // Wait until editor is focused to join the peer to peer network.
+        this.$editable[0].addEventListener('focus', this._joinPeerToPeer);
```

Comment:
Aren't there cases where we might reset the editor while not having focused on it the first time and that way we just stack up the `focus` event listeners and then multiple listeners would be called at the same time when we focus the first time ? Like switching reccords with the arrows for example.

Reply:
Damn yes I guess you're right. We could add a removeListener at the beginning over resetEditor, I think that should cover that.

Reply:
It might be interesting to try it first to see if the issue really occurs or if there is some mechanisms that interfere and that I didn't think about initially. For example, maybe it's not the same `editable` node in practice ?